### PR TITLE
add filter to cli

### PR DESCRIPTION
**Pull request by lasshpanzer**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: client/volume_test.go [hunk 1]
```diff
@@ -190,7 +190,7 @@ func TestListVolumes(t *testing.T) {
 		},
 	}
 
-	vols, err := fv.ListVolumes()
+	vols, err := fv.ListVolumes(map[string]string{"limit": "3", "offset": "4"})
```
Comment (leonwanghui) [inline client/volume_test.go]: I suggest that the returned `vols` be asserted using `reflect.DeepEqual()`

File: openapi-spec/swagger.yaml [hunk 2]
```diff
@@ -1446,27 +1396,32 @@ definitions:
...
+            readOnly: true
+          RemoveVolumes:
             type: array
             items: 
               type: string
-            example: 
-              - 993c87dc-1928-498b-9767-9da8f901d6ce 
-              - 90d667f0-e9a9-427c-8a7f-cc714217c7bd
+            readOnly: true
+          GroupSnapshots:
```
Comment (xing-yang) [inline openapi-spec/swagger.yaml]: GroupSnapshots have not been implemented yet.  Why are they added now?

Comment (lasshpanzer) [reply openapi-spec/swagger.yaml]: OK

**Comment by stmcginnis** [comment]
In future PRs, please add at least basic information in the PR description to fill in the template provided.

**Pull request merged**